### PR TITLE
Add libvirt usb utils file

### PR DESCRIPTION
**Pull request by chunfuwen**
Add libvirt usb utils file

There are some common utils to manipulate usb devices,
so pull them into one indepentent file here

Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
test results can be found from #https://github.com/autotest/tp-libvirt/pull/4444

File: virttest/utils_libvirt/libvirt_usb_utils.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+"""
```
Comment (dzhengfy) [inline virttest/utils_libvirt/libvirt_usb_utils.py:1]: For the file name, let's align most files's style in this directory and make it shorter, like 'libvirt_usb.py', ok?

File: virttest/utils_libvirt/libvirt_usb_utils.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+from avocado.utils import process
+
+LOG = logging.getLogger('avocado.' + __name__)
+
+
+def get_usbs_lists_on_host():
+    """
+    This function gets usb devices located
+    on host system as a list
+    """
```
Comment (dzhengfy) [inline virttest/utils_libvirt/libvirt_usb_utils.py:19]: Add docstring for return value

File: virttest/utils_libvirt/libvirt_usb_utils.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    :param session: a login session to VM
+    :param partition: The disk partition in VM to be checked.
+    :return: If the disk is a usb device, return True.
+    """
+    try:
+        cmd = "ls -l /dev/disk/by-id/ | grep %s | grep -i usb" % partition
+        status = session.cmd_status(cmd)
+        return status == 0
+    except Exception as err:
+        logging.debug("Error happens when check if new disk is usb device: %s",
```
Comment (dzhengfy) [inline virttest/utils_libvirt/libvirt_usb_utils.py:40]: It seems better to use `logging.error`, right?

**Review by dzhengfy — COMMENTED** [review]
Others LGTM

File: virttest/utils_libvirt/libvirt_usb_utils.py (hunk 1 above)
Comment (Yingshun) [reply virttest/utils_libvirt/libvirt_usb_utils.py:1]: +1

File: virttest/utils_libvirt/libvirt_usb_utils.py (hunk 3 above)
Comment (Yingshun) [reply virttest/utils_libvirt/libvirt_usb_utils.py:40]: It should be LOG.error, r8?

File: virttest/utils_libvirt/libvirt_usb_utils.py (hunk 1 above)
Comment (chunfuwen) [reply virttest/utils_libvirt/libvirt_usb_utils.py]: @dzhengfy 
@Yingshun 
updated

File: virttest/utils_libvirt/libvirt_usb_utils.py (hunk 2 above)
Comment (chunfuwen) [reply virttest/utils_libvirt/libvirt_usb_utils.py]: @dzhengfy 
updated

File: virttest/utils_libvirt/libvirt_usb_utils.py (hunk 3 above)
Comment (chunfuwen) [reply virttest/utils_libvirt/libvirt_usb_utils.py]: @dzhengfy 
updated to LOG.error

File: virttest/utils_libvirt/libvirt_usb.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    :param session: a login session to VM
+    :param partition: The disk partition in VM to be checked.
+    :return: If the disk is a usb device, return True.
+    """
+    try:
+        cmd = "ls -l /dev/disk/by-id/ | grep %s | grep -i usb" % partition
+        status = session.cmd_status(cmd)
+        return status == 0
+    except Exception as err:
+        LOG.debug("Error happens when check if new disk is usb device: %s",
```
Comment (Yingshun) [inline virttest/utils_libvirt/libvirt_usb.py:42]: It's still in 'debug' level, please see @dzhengfy 's comment in https://github.com/avocado-framework/avocado-vt/pull/3499#discussion_r957946752

Comment (chunfuwen) [reply virttest/utils_libvirt/libvirt_usb.py]: @Yingshun 
updated, thanks

**Review by Yingshun — APPROVED** [review]

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**